include/core/Partition_Data.hpp

```diff
@@ -0,0 +1,14 @@
+#ifndef PARTITION_DATA_H
+#define PARTITION_DATA_H
+
+using Tuple = std::tuple<int, std::string, std::string, std::string>;
+
+struct PartitionData
+{
```

Comment:
```suggestion
struct PartitionData
{
    using Tuple = std::tuple<int, std::string, std::string, std::string>;
```
Don't declare a global type named `Tuple`

Reply:
I tried to make the change. But it caused some build errors in ```Partition_Parser.hpp```. Apparently, ```Partition_Parser.hpp``` uses this declaration in a few places in its code region. In its current form, the ```Partition_Parser.hpp``` would be the same as its original form in master branch.

Reply:
It would become `PartitionData::Tuple`.

Reply:
Right. Implemented.